tests/Microsoft.Identity.Test.Unit/CoreTests/HttpTests/HttpManagerTests.cs

```diff
@@ -135,7 +135,7 @@ public void TestSendGetWithCanceledToken()
                 catch (AggregateException exc)
                 {
                     Assert.IsNotNull(exc);
-                    Assert.IsTrue(exc.InnerException is MsalServiceException);
+                    Assert.IsTrue(exc.InnerException is TaskCanceledException);
```

Comment:
Please make this test `async` and use `await httpManager.SendGetAsync` to avoid this whole AggregateException stuff. And maybe add a comment on the test that this is a regression test for bug xyz.

Reply:
Done.